lib/stepmod/utils/express_bibdata.rb

```diff
@@ -0,0 +1,112 @@
+module Stepmod
+  module Utils
+    class ExpressBibdata
+      DOCNUMBER = 10303
+
+      attr_accessor *%w(
+        type doctype part title_en version pub_year pubid published_info
+      )
+
+      def initialize(schema:)
+        # module, resource, application_protocol, business_object_model
+        # @type = document.name
+
+        # raise "UnknownFileError" unless %w(module resource
+        #                                    application_protocol business_object_model).include?(@type)
+
+        @published_info = extract_remarks(schema, "__published_in")&.first
```

Comment:
How about move extraction of Annotated EXPRESS remarks (e.g. `schema.entity.id.__xxx:name`) into Expressir?

Like this:
```ruby
# (*"schema.__published_in" ... *)`
Expressir::Express::Parser.from_file(express_file).schemas.first.remarks("__published_in").first
=> content of `__published_in`

# (*"schema.entity.__note" ... *)`
Expressir::Express::Parser.from_file(express_file).schemas.first.entities.first.remarks("__note").first
=> content of `__note`

# Direct fetch of a named remark
# (*"schema.entity.__note:note1" ... *)`
Expressir::Express::Parser.from_file(express_file).schemas.first.entities.first.remarks(name: "note1")
=> content of `__note:note1`
```

Reply:
This does not hold up this PR, though.

Reply:
Just found out that there is a find method in exprssir that we can use like schema.find("__published_in"). Updated to use the find method in place of the extract_remarks method.